<launch>
    <node pkg="lane_detection" type="ulfast.py" name="lane_detection_node" output="screen"/>
</launch>
